### ISSUE TITLE
Use custom model/VAE path if chosen

### ISSUE THREAD

**Issue by arcum42** [Zetaphor/sd-library-notes#8]
Looking at another extension's code,  model toolkit, I can see that they find the model and vae path with:
MODEL_SAVE_PATH = shared.cmd_opts.ckpt_dir or os.path.join("models", "Stable-diffusion")
VAE_SAVE_PATH = shared.cmd_opts.vae_dir or os.path.join("models", "VAE")

We can probably do the same. Just felt lazy and made an issue instead of a pr. :P

https://github.com/arenatemp/stable-diffusion-webui-model-toolkit

**Comment by arcum42** [comment]
Now having actually had a chance to try setting ckpt_dir, it looks like it loads the models there *in addition* to the normal ones, if they exist. (You set it in webui-user.sh/bat, in the command line arguments.)

It also seems like besides ckpt_dir and vae_dir, there's embeddings_dir, hypernetwork_dir, & lora_dir. The current model is in ckpt, btw. (Doing "print(shared.cmd_opts)" gets you the full list.)

I'm thinking we can check to see if each of those variables have anything in them, and scan them if they do, but *also* scan the base directories.

**Comment by Zetaphor** [comment]
I'll look into it, currently dealing with both of my extensions (and others I rely on) being broken with the latest Automatic updates lol

**Comment by arcum42** [comment]
Yeah, commented before that last update wrecked havoc on it. Not sure how much will have to be fixed on the extension side and how much on the A1111 side, too, since they did a big gradio update...

**Comment by arcum42** [comment]
@Eugenii10 
That'd be a separate idea, and should be a separate issue. This issue is about the fact that right now, the extension assumes all the models and such are in the default places, so if you've specified different places by command line arguments, it'll ignore that.

Auto selecting the notes for a model when you choose it would be nice, it's just a different issue.
